### PR TITLE
wdio-config: include and exclude at capability level

### PR DESCRIPTION
**Pull request by ergbouncex**
## Proposed changes

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

Allowing spec include/exclude at the capability level to filter properly while passing multiple suites of tests. Previously the capability include/exclude only worked if no suites were passed.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments
Fixes issue #3925

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### Reviewers: @webdriverio/technical-committee

### CODE REVIEW

File: packages/wdio-config/tests/configparser.test.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ describe('ConfigParser', () => {
             const specs = configParser.getSpecs()
             expect(specs).toContain(__filename)
             expect(specs).toContain(INDEX_PATH)
-            expect(specs).toContain(path.join(__dirname, 'validateConfig.test.js'))
+            expect(specs).not.toContain(path.join(__dirname, 'validateConfig.test.js'))
```
Comment (mgrybyk) [inline packages/wdio-config/tests/configparser.test.js]: Why do we need to invert this assertion?

File: packages/wdio-config/src/lib/ConfigParser.js [hunk 2]
```diff
@@ -174,9 +175,18 @@ export default class ConfigParser {
...
+                tmpSpecs = tmpSpecs.concat(ConfigParser.getFilePaths(capSpecs))
+            }
+
+            if (Array.isArray(capExclude)) {
+                exclude = exclude.concat(ConfigParser.getFilePaths(capExclude))
+            }
 
-            return [...new Set(tmpSpecs)]
+            specs = [...new Set(tmpSpecs)]
+            return specs.filter(spec => exclude.indexOf(spec) < 0)
```
Comment (mgrybyk) [inline packages/wdio-config/src/lib/ConfigParser.js]: Can be replaced with !exclude.inlcudes(spec)

File: packages/wdio-config/tests/configparser.test.js (hunk 1 above)
Comment (ergbouncex) [reply packages/wdio-config/tests/configparser.test.js]: The original assertion was not actually checking the _exclude_ defined within FIXTURES_CONF which has:
```
    exclude: [
        path.join(TEST_ROOT, '/validateConfig.test.js')
    ]
```

File: packages/wdio-config/src/lib/ConfigParser.js (hunk 2 above)
Comment (ergbouncex) [reply packages/wdio-config/src/lib/ConfigParser.js]: Sure. The same syntax is used outside of the suite loop too. I will update both

**Pull request merged**